db/data/20220929151206_populate_room_limit_permission.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class PopulateRoomLimitPermission < ActiveRecord::Migration[7.0]
+  def up
+    Permission.create! [
+      { name: 'roomLimit' }
```

Comment:
Could we name it RoomLimit with a capital R for consistency

Reply:
Done